I get the following error while testing usage.
```
Error		CMake Error at F:/boinc/installed/x64-windows/share/boinc/boinc-config.cmake:60 (set_target_properties):
  The link interface of target "unofficial::boinc::boinc" contains:

    OpenSSL::SSL

  but the target was not found.  Possible reasons include:

    * There is a typo in the target name.
    * A find_package call is missing for an IMPORTED target.
    * An ALIAS target is missing.		F:/boinc/installed/x64-windows/share/boinc/boinc-config.cmake	60	
```

--- a/ports/boinc/fix-build.patch
+++ b/ports/boinc/fix-build.patch
@@ -1,0 +1,49 @@
+diff --git a/lib/filesys.cpp b/lib/filesys.cpp
+index c258a0f8d1..76530fb432 100644
+--- a/lib/filesys.cpp
++++ b/lib/filesys.cpp
+@@ -63,6 +63,7 @@
+ 
+ #include "error_numbers.h"
+ #include "filesys.h"
++#include "str_replace.h"
+ #include "str_util.h"
+ #include "util.h"
+ 
+diff --git a/lib/str_util.h b/lib/str_util.h
+index 0e2d5f179a..22908d8568 100644
+--- a/lib/str_util.h
++++ b/lib/str_util.h
+@@ -22,8 +22,6 @@
+ #include <vector>
+ #include <string.h>
+ 
+-#include "str_replace.h"
+-
+ #define safe_strcpy(x, y) strlcpy(x, y, sizeof(x))
+ #define safe_strcat(x, y) strlcat(x, y, sizeof(x))
+ 
+diff --git a/lib/util.cpp b/lib/util.cpp
+index cfa495a6f9..2af4b0fb94 100644
+--- a/lib/util.cpp
++++ b/lib/util.cpp
+@@ -17,6 +17,7 @@
+ 
+ #if defined(_WIN32)
+ #include "boinc_win.h"
++#include "str_replace.h"
+ #include "str_util.h"
+ #include "win_util.h"
+ #endif
+diff --git a/lib/win_util.cpp b/lib/win_util.cpp
+index 86289e0ce7..82b17f4425 100644
+--- a/lib/win_util.cpp
++++ b/lib/win_util.cpp
+@@ -23,6 +23,7 @@
+ #include "util.h"
+ #include "filesys.h"
+ #include "win_util.h"
++#include "str_replace.h"
+ #include "str_util.h"
+ 
+ /**

--- a/ports/boinc/portfile.cmake
+++ b/ports/boinc/portfile.cmake
@@ -6,6 +6,8 @@ vcpkg_from_github(
     REF client_release/7.22/7.22.1
     SHA512 8b3efc68fe8df8f95a674d9deddbe355da2070a960b1768ee7f9c2afdd8a373e97297715dec5d7daf131d4b5c478afbc4476e152ec516080620f66e1a1f785af
     HEAD_REF master
+    PATCHES
+        fix-build.patch
 )
 
 file(COPY ${CMAKE_CURRENT_LIST_DIR}/CMakeLists.txt DESTINATION ${SOURCE_PATH})
@@ -37,6 +39,12 @@ vcpkg_cmake_configure(
 vcpkg_cmake_install()
 
 vcpkg_cmake_config_fixup()
+file(READ "${CURRENT_PACKAGES_DIR}/share/boinc/boinc-config.cmake" BOINC_CONFIG)
+file(WRITE "${CURRENT_PACKAGES_DIR}/share/boinc/boinc-config.cmake" "
+include(CMakeFindDependencyMacro)
+find_dependency(OpenSSL)
+${BOINC_CONFIG}
+")
 
 vcpkg_copy_pdbs()
 

--- a/ports/boinc/vcpkg.json
+++ b/ports/boinc/vcpkg.json
@@ -1,6 +1,7 @@
 {
   "name": "boinc",
   "version": "7.22.1",
+  "port-version": 1,
   "description": "Open-source software for volunteer computing and grid computing.",
   "homepage": "https://boinc.berkeley.edu/",
   "license": "LGPL-3.0-or-later",

--- a/versions/b-/boinc.json
+++ b/versions/b-/boinc.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "0d3f7220d72e02720ad44a15cfc46ebc1a6c26b4",
+      "version": "7.22.1",
+      "port-version": 1
+    },
+    {
       "git-tree": "05fdc41be633efa59debd81ec8a95c584972e54b",
       "version": "7.22.1",
       "port-version": 0

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -610,7 +610,7 @@
     },
     "boinc": {
       "baseline": "7.22.1",
-      "port-version": 0
+      "port-version": 1
     },
     "bond": {
       "baseline": "10.0.0",

